demos/mosquitto/snapcraft.yaml

```diff
@@ -19,13 +19,13 @@ apps:
 
 parts:
   mosquitto:
-    plugin: copy
+    source: conf
+    plugin: dump
```

Comment:
Is there a reason the plugin is now after source? Should the other demos be updated to look the same?